Kitodo-API/src/main/java/org/kitodo/api/dataformat/LinkedStructure.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataformat;
+
+import java.math.BigInteger;
+import java.net.URI;
+
+/**
+ * A link to a structure in a different {@code Workpiece}.
+ */
+public class LinkedStructure implements ExistingOrLinkedStructure {
+
+    /**
+     * The label of the linked structure. The label is saved in the linked file.
+     */
+    private String label;
+
+    /**
+     * The order is a number that serves to place other structures together with
+     * them in the correct order in the same parental structure. After a link
+     * has been made, the order value no longer affects the order. The order is
+     * saved in the file that contains the link.
+     */
+    private BigInteger order;
+
+    /**
+     * The type of structure, for example, book, chapter, page. Although the
```

Comment:
Seems like an ENUM is required here

Reply:
The type of linked structure is the name of the division from the ruleset. Therefore, no `enum` is possible here because the value is set in the ruleset, not in the source code.

Reply:
Are those values not constrained in some way? Or can I also write there 'potato' and it will work?

Reply:
Yes, if you want to digitize a [seed bank](https://en.wikipedia.org/wiki/Seed_bank) ...

Reply:
So why do you point out to use a controlled vocabulary? This type should only be set from values, read from the ruleset, so the "controlled vocabulary" should be defined there. A programmer should not care about it.
Maybe you should mention, that this value can only be set from ruleset values.

Reply:
Why do I mention that? So that a programmer who is new to the project understands these relationships; that the value here should be an element of a limited (though not fixed in code) set of strings.

It's not entirely true that the value can only be set by the ruleset. The read function also retains values that exist in the METS file, even if they are not in the ruleset. They are then highlighted in the frontend. This is so that you can open the METS file at all and then change either the ruleset or the METS file. This is better than Production 2.x, which simply crashed when an element in the ruleset was undefined. If you wanted to delete the wrong element from the METS file, you first had to define it in the ruleset to be able to open the file. That is not necessary anymore.